learning/tour-of-beam/terraform/cloud_functions/variables.tf

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+variable "service_account_id" {
+  description = "The name of Service Account to run Cloud Function"
```

Comment:
This could probably just be defaulted or put in a common.tfvars file.

Reply:
Prefix concept and random string has been implemented to generate some variable values
All the variables' values that are not import for the manual input are generated automatically using prefix and random string of 4 symbols